src/BubbleGroup/index.tsx

```diff
@@ -4,36 +4,42 @@ import DefaultChatBubble from "../ChatBubble";
 import styled from "styled-components";
 
 const BubbleGroupContainer = styled.div`
-  margin: 10px 0;
+  margin: 16px 0;
   overflow: auto;
   position: relative;
 `;
 
-const BubbleGroupHeader = styled.h5`
-  margin: 0;
-  font-size: 14px;
-  font-weight: bold;
-  color: #999;
-  font-family: ${props => props.theme.fontFamily};
-`;
-
 export default function BubbleGroup(props: BubbleGroupInterface) {
-  const { showSenderName, chatBubble, senderName, messages } = props;
+  const { chatBubble, senderName, messages } = props;
   const ChatBubble = chatBubble || DefaultChatBubble;
   const sampleMessage = messages[0];
 
+  const senderNameHandler = () => {
```

Comment:
There doesn't seem to be an obvious reason to have this as a function. Can you just do like:

```
let senderNameToRender = ""
if (senderName || sampleMessage.senderName) {
  senderNameToRender = ...
}
```